Eagle-Rock-Analytics/historical-obs-platform
# Updates the temp_master_cleaned_stationlist for cleaned variable coverage

Updates the 2_clean_stationvisualizer notebook function that generates the station list of all networks all stations to now include the clean variable coverage stats. 

Presently, CWOP and RAWS are the only two outstanding networks that need to be run before that station list is 100% updated, so once those two networks are in this notebook will be ready to be run again; but pushing this now as able to provide some precipitation coverage stats for precip localization (and qaqc function development!)

----
**To test**
Either check out the updated stationlist that is in our cleaned bucket; or run this notebook (its cell 3 that generates it) with the update option turned on to True to regnerate this list
